components/map/selectors.js

```diff
@@ -426,43 +440,82 @@ export const getActiveLayers = createSelector(
 
 export const getActiveLayersWithDates = createSelector(
   getActiveLayers,
-  layers => {
+  (layers) => {
     if (isEmpty(layers)) return [];
-    return layers.map(l => {
+    return layers.map((l) => {
       const { decodeFunction, decodeParams } = l;
       const { startDate, endDate } = decodeParams || {};
+      const { layerConfig } = l;
+      const { params_config = [] } = layerConfig;
+
+      // Some filters requires values to be spread
+      // this modifies the filters that has those requirements to return all values
+      const requiresSpread = params_config.filter((config) =>
```

Comment:
Nice solution - I guess the question is should we flag this in the `params_config` on the layer with `is_spread_key: true` or something?

In the mapbox vector styles regime in theory it will always  be  a string or an array that we pass in to the render... so maybe this solution is enough.

Opinions, @edbrett ?